src-cpp/app/modules/shared/section_item.h

```diff
@@ -39,15 +40,16 @@ class SectionItem : public QObject
     Q_PROPERTY(bool ensOnly READ getIsEnsOnly)
 
 public:
-    SectionItem(QString id,
-                SectionType sectionType,
-                QString name,
-                QString description,
-                QString image,
-                QString icon,
-                QString color,
+    SectionItem(QObject* parent = nullptr,
+                QString id = "",
+                SectionType sectionType = SectionType::Unkown,
+                QString name = "",
+                QString description = "",
+                QString image = "",
+                QString icon = "",
+                QString color = "",
```

Comment:
@Khushboo-dev-cpp sorry I think you missed that place :)

Reply:
all done and merged , thanks @borismelnik !